draft-irtf-cfrg-hpke.md

```diff
@@ -183,7 +183,7 @@ The following terms are used throughout this document to describe the
 operations, roles, and behaviors of HPKE:
 
 - Initiator (I): Sender of an encrypted message.
```

Comment:
Do we want to change Initiator (I) to Sender (S)?
If so, do we want to change I everywhere (e.g. pkI) to S (i.e. pkS)?
Its a biggish change, so we'd better all agree.

Reply:
Yep, I think so. I applied that suggestion. Please have a look!